go.mod

```diff
@@ -157,3 +150,6 @@ replace github.com/openshift/api => github.com/openshift/api v0.0.0-202108310919
 
 // Required until the following jira is resolved - https://issues.redhat.com/browse/MGDAPI-4996
 replace github.com/grafana-operator/grafana-operator/v4 => github.com/grafana-operator/grafana-operator/v4 v4.2.0
+
+// Required until 3scale PR merge https://github.com/3scale/3scale-operator/pull/854
+replace github.com/3scale/3scale-operator => github.com/valerymo/3scale-operator v0.0.0-20230831101242-4e5f852570a0
```

Comment:
Think we can remove this replace. Should only be in your local branch not in the PR

Reply:
As the 3scale pr is merged we can definitely remove this.

Reply:
done. Thank you